NSubstitute.DbConnection/NSubstitute.DbConnection.Tests/QueryTests.cs

```diff
@@ -7,21 +7,30 @@
 
 namespace NSubstitute.DbConnection.Tests;
 
+using System.Data;
 using System.Data.Common;
 
 public class QueryTests
 {
-    [Test]
-    public void ShouldMockQuery()
+    private const string SimpleSelect = "select * from table";
+
+    private static DbCommand GetSimpleCommandSingleRow(Action<DbConnection> open)
```

Comment:
Can we move this to the end of the class - I know I don't have method order linting enabled but prefer public before private